gvsbuild/projects/ffmpeg.py

```diff
@@ -69,7 +69,7 @@ def __init__(self):
         Project.__init__(
             self,
             "nv-codec-headers",
-            version="11.1.5.2",
+            version="12.0.16.0",
```

Comment:
this is tricky, this means that we are bumping the required nvidia driver that we need. We should probably keep the latest 11.x version for now

Reply:
Thanks @nacho, I dropped that commit :+1: